tests/src/test/scala/system/utils/KafkaUtils.scala

```diff
@@ -66,26 +66,27 @@ trait KafkaUtils extends TestHelpers with WskTestHelpers {
         new RestAssuredConfig().sslConfig(config)
     }
 
-    def createTrigger(assetHelper: AssetCleaner, name: String, parameters: Map[String, spray.json.JsValue]) = {
+    def createTrigger(assetHelper: AssetCleaner, name: String, parameters: Map[String, spray.json.JsValue]): String = {
         println(s"Creating trigger $name")
 
         val feedCreationResult = assetHelper.withCleaner(wsk.trigger, name) {
             (trigger, _) =>
                 trigger.create(name, feed = Some(s"/whisk.system/messaging/messageHubFeed"), parameters = parameters)
         }
 
-        withActivation(wsk.activation, feedCreationResult, initialWait = 5 seconds, totalWait = 60 seconds) {
```

Comment:
Don't need to query for the activation as we already have it from trigger create.